package.json

```diff
@@ -12,6 +12,12 @@
   "mocha": {
     "config": "spec/.mocharc.json"
   },
+  "browserslist": [
+    "> 2%",
+    "last 2 versions",
+    "ie 11",
+    "not dead"
+  ],
```

Comment:
Why did you put this here and not in the `.browserslistrc` file?

Reply:
I was following the guidance based on this:
https://github.com/browserslist/browserslist#queries
> 1. browserslist key in package.json file in current or parent directories. We recommend this way.
>  1. .browserslistrc config file in current or parent directories.
>  1. browserslist config file in current or parent directories.
>  1. BROWSERSLIST environment variable.
>  1. If the above methods did not produce a valid result Browserslist will use defaults: > 0.5%, last 2 versions, Firefox ESR, not dead.

Reply:
great, thanks